ha/util/message_bus.py

```diff
@@ -131,19 +139,48 @@ def run(self):
             except Exception as e:
                 Log.error(f"Supressing exception from message bus {e}")
                 retry = False
+        if self.flush_on_exit:
```

Comment:
Can you please add a comment here saying that we do not expect any messages to be present in the message bus at this point ( since previously received cluster stop would have ensured that message bus is empty) and this is just a precautionary code.

Reply:
done